Sources/Security/Signing+ResponseVerification.swift

```diff
@@ -95,3 +94,16 @@ extension HTTPResponse where Body == Data? {
     }
 
 }
+
+private extension HTTPRequest {
+
+    var requiresSignature: Bool {
```

Comment:
Should this also check if entitlement verification is enabled?

Reply:
That's done by the calling method. But I removed this property so it's more clear now.